src/lib/runtime_config/runtime_config.ml

```diff
@@ -1,9 +1,21 @@
 open Core_kernel
 
 module Fork_config = struct
+  (* previous_global_slot here is a global slot since genesis. previous
```

Comment:
comment bitrot a bit (previous_global_slot renamed)

Reply:
can you inline these comments for each field, that will make it easier to notice when they change?

Reply:
Done in [a681c0f](https://github.com/MinaProtocol/mina/pull/14285/commits/a681c0f0d61834e0edd959fcc52ff01525d8cd50).